components/SnippetDetails.js

```diff
@@ -0,0 +1,36 @@
+import Link from "next/link";
+import { useRouter } from "next/router";
+import useSWR from "swr";
+
+function SnippetDetails() {
+  const router = useRouter();
+
+  const { id } = router.query;
+
+  const { data, isLoading, error } = useSWR(`/api/snippets/${id}`);
+
+  if (error) return <div>failed to load Details View 🥺</div>;
+  if (isLoading) {
+    return <h1>Loading Details View 🤓</h1>;
+  }
+  if (!data) {
+    return <div>cant find the data yet</div>;
+  }
+  const { name, code, description, link, tags } = data;
+
+  return (
+    <section>
+      <h1>{name}</h1>
```

Comment:
this will lead to you having two h1 on your page. One from your SnippetDetails component and one from your Header component.

I would change this to an h2